tests/ct_transaction.c

```diff
@@ -0,0 +1,158 @@
+#include "hircluster.h"
+#include <assert.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define CLUSTER_NODE "127.0.0.1:30001"
+
+// Test of transactions in using the pipelined sync api
+void test_pipelined_transaction() {
+    redisClusterContext *cc = redisClusterContextInit();
+    assert(cc);
+    redisClusterSetOptionAddNodes(cc, CLUSTER_NODE);
+    redisClusterConnect2(cc);
+
+    assert(cc->err == 0);
+
+    int status;
+    status = redisClusterAppendCommand(cc, "MULTI");
+    assert(status == REDIS_OK);
+    status = redisClusterAppendCommand(cc, "SET bar five");
+    assert(status == REDIS_OK);
+    status = redisClusterAppendCommand(
+        cc, "SET foo five"); // Not ok due to other slot
+    assert(status != REDIS_OK);
+    status = redisClusterAppendCommand(cc, "GET bar");
```

Comment:
I think it aligns better with the rest of the code like this:

```C
    status = redisClusterAppendCommand(cc, "SET foo five");
    assert(status != REDIS_OK); // Not ok due to other slot
```

Reply:
:+1: